### PR TITLE
DL-12652 play 3.0 migration

### PR DESCRIPTION
**Pull request by pankaj2453**
# Pull Request Checklist
* [ ] Branch is rebased with master
* [ ] Added Unit Tests for changed functionality
* [ ] Ran `sbt clean compile coverage test it:test coverageReport`, all tests pass and coverage meets minimum
* [ ] Checked libraries are up to date with `sbt validate`. (If applicable) library updates are done in a separate, single commit
* [ ] Ran Acceptance tests as per the story. Query in group slack if unsure of test packs to run.
* [ ] Ran Performance tests as per the story. Query in group slack if unsure of test packs to run.
* [ ] (If applicable) raised app-config-* PRs
* [ ] (If applicable) added Integration Test

### CODE REVIEW

**Review by danny-waughman — DISMISSED** [review]

File: project/plugins.sbt [hunk 1]
```diff
@@ -20,12 +20,12 @@ resolvers += Resolver.url("HMRC-open-artefacts-ivy2", url("https://open.artefact
...
+addSbtPlugin("uk.gov.hmrc"        % "sbt-auto-build"          % "3.22.0")
 
-addSbtPlugin("uk.gov.hmrc"        % "sbt-distributables"      % "2.2.0")
+addSbtPlugin("uk.gov.hmrc"        % "sbt-distributables"      % "2.5.0")
 
-addSbtPlugin("org.scalastyle"     %% "scalastyle-sbt-plugin"  % "1.0.0")
+addSbtPlugin("org.scalastyle"     %% "scalastyle-sbt-plugin"  % "1.0.0" exclude("org.scala-lang.modules", "scala-xml_2.12"))
 
-addSbtPlugin("org.scoverage"      % "sbt-scoverage"           % "1.9.3")
+addSbtPlugin("org.scoverage"      % "sbt-scoverage"           % "2.0.9")
```
Comment (derickdamoah) [inline project/plugins.sbt:29]: this can go up to 2.0.12

File: build.sbt
Comment (derickdamoah) [inline build.sbt:1]: can you update the scalatest dependency to "org.scalatestplus" %% "mockito-4-11" % "3.2.18.0" % Test

Also can you move the integration tests to their own folder and modify the readme?

File: project/plugins.sbt (hunk 1 above)
Comment (pankaj2453) [reply project/plugins.sbt]: updated

File: build.sbt
Comment (pankaj2453) [reply build.sbt:1]: > can you update the scalatest dependency to "org.scalatestplus" %% "mockito-4-11" % "3.2.18.0" % Test
> 
> Also can you move the integration tests to their own folder to comply with the [migration guide](https://docs.google.com/document/d/1LpahhQNmb7XMqOFp0_aAnKgrnTzA26XwdwpceQdx_ak/edit) and modify the readme?

updated

**Pull request merged**